owl/EASE-STATE.owl

```diff
@@ -219,27 +233,7 @@ Note that this State focuses on how the interaction of, usually non-agentive, ob
                 <owl:allValuesFrom rdf:resource="http://www.ease-crc.org/ont/EASE-STATE.owl#Gestallt"/>
             </owl:Restriction>
         </rdfs:subClassOf>
-    </owl:Class>
-    
-
-
-    <!-- http://www.ease-crc.org/ont/EASE-STATE.owl#LinkageConfiguration -->
-
-    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-STATE.owl#LinkageConfiguration">
```

Comment:
Linkage is gone because we no longer put the assembly stuff in the EASE ontos, right?